cmd/env.go

```diff
@@ -56,6 +62,24 @@ func (e *Environment) Branches() (branches []string) {
 	return
 }
 
+func (e *Environment) Batches() map[string][]string {
+	lines := strings.Split(e.InputBatches, "\n")
+	m := make(map[string][]string, len(lines))
+	for _, line := range lines {
+		line = strings.TrimSpace(line)
```

Comment:
```suggestion
```

Prefixes are trimmed by `strings.TrimSpace(split[0])` on :77
Suffixes are trimmed by the final iteration on :75.
Skippit :shipit: .